packages/reporter/src/commands/command.tsx

```diff
@@ -259,53 +265,61 @@ class Command extends Component<Props> {
     const isSessionCommand = commandName === 'session'
     const displayNumOfChildren = !isSystemEvent && !isSessionCommand && model.hasChildren && !model.isOpen
 
+    let groupPlaceholder: Array<JSX.Element> = []
```

Comment:
This can be a const right?

```suggestion
    const groupPlaceholder: Array<JSX.Element> = []
```